content/md/productivity/what-is-a-project_fr

```diff
@@ -0,0 +1,33 @@
+---
+id: what-is-a-project
+themes: boost-your-productivity, collaborative-work
+title: Qu'est-ce qu'un **projet**?
+popular: true
+ee-only: true
+related: manage-your-projects, projects-notifications
+---
+
+# Introduction
+Un projet est une fonctionnalité de Teamwork Assistant. C'est une nouvelle façon d'organiser vos travaux autour d'une activité spécifique. Il va vous aider à organiser votre travail personnel ou en équipe. 
+
+::: warning
```

Comment:
This line shouldn't be here. It should be on line 20.

Reply:
Hi Nolwenn, can you tell me how to edit  ? 

Thank you

Reply:
Hi Julie!
Just saw that you deleted the "what-is-a-project_fr" file from this PR, which is good since we try to keep one PR per article.
You can now create a new PR with the translation of this article in it. You'll be able to move the line I talked about above, to the right place.
If I'm not clear enough, we can make a quick call so that I can help you. Don't hesitate to ask!
Thanks!